src/main/java/teammates/storage/api/FeedbackSessionsDb.java

```diff
@@ -45,20 +45,16 @@
 
         List<FeedbackSession> endEntities = load()
                 .filter("endTime >", curStart)
-                .filter("endTime <=", curEnd)
                 .list();
```

Comment:
from the discussions in #8421, I think  we should use key-only query, i.e. `.keys()`

Reply:
Yup this will be much more efficient. Use `.keys().list()` to get a list of keys. After getting both lists of keys and filtering for the relevant keys, run a batch get to fetch the actual `FeedbackSession` entities: `ofy().load().keys(listOfRelevantKeys).values()`.